### DISCUSSION TITLE
5.2.0 and TypeScript

### DISCUSSION THREAD

**Discussion by luigigorlero** [Intevel/nuxt-directus#145] [General]
After `5.2.0`, all my requests that use `getItems`, which I have been writing like so:
```
const articles = await getItems<Article>({
  collection: 'articles',
  params: { fields: ['*'] },
})
```
now issue a TypeScript warning. If i want to access any property, I get the ominous `property does not exist` since the variable is now a union type `const articles: DirectusItems<Article> | Article[]` instead of simply `Article[]`

What are your best practices in this regard?

**Comment by ranseur92** [comment]
Fixed in https://github.com/Intevel/nuxt-directus/pull/146
